org-code-javabuilder/lib/src/main/java/org/code/javabuilder/ClientMessageType.java

```diff
@@ -0,0 +1,11 @@
+package org.code.javabuilder;
+
+/** Message keys that map 1:1 to client-side usages. */
+public enum ClientMessageType {
+  /** A message directed to the client-side terminal. Equivalent to System.out.print. */
+  systemOut,
+  /** An exception that should be displayed to the user. */
+  exception,
+  /** A neighborhood signal that directs the client-side Neighborhood program to take an action. */
+  neighborhood
```

Comment:
We'll send a stream of events. Maze animates the events as they happen and maintains the state for the UI.

Reply:
Ok, this is probably a discussion outside of this PR, but I'm curious about:
1) If you're worried that we'll end up with duplicated Neighborhood logic on the client and server.
2) If you're worried about the client and server falling out of sync.  (For example, with HTTP, it might be easy to lose events or receive them out of order.)